shared-libs/search/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "search",
+  "version": "1.0.0",
+  "description": "Our implemenation of search using (C|P)ouchDB views.",
+  "main": "src/search.js",
+  "scripts": {
+    "test": "./node_modules/.bin/mocha ./test"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "chai": "^4.1.2",
+    "mocha": "^5.0.0",
+    "sinon": "4.0.1"
+  },
+  "dependencies": {
+    "moment": "2.19.1",
+    "underscore": "1.8.3"
```

Comment:
How strict do we want/need to be with dependencies in the shared lib?  Presumably this will impact any `underscore`, `moment` dependency versions in webapp, and therefore affect what might be bundled...

Reply:
Yeah, AFAICT we need to be very strict, hence the explicit versions. I am wondering if the best thing to do is to actually copy those sections out of the `medic-webap/package-lock.json` to make sure they are 100% the same